R/calc_AUC.R

```diff
@@ -0,0 +1,99 @@
+require(caTools)
+
+calc_AUC <-
+  function(qp,
+           test_matrix_Xi,
+           test_matrix_Xip,
```

Comment:
it would be great if we could also use this function to compute AUC for the ELO model. To do that I suggest you remove these first 3 arguments, and replace them with a single test_rank_difference argument (a vector of rank differences from either the SVM or ELO)